### PR TITLE
UF-7948: Move messagetype to string value and add new valid value

### PR DESCRIPTION
**Pull request by cheezi0747**

### CODE REVIEW

File: src/main/java/se/sundsvall/casedata/api/model/validation/enums/MessageType.java [hunk 1]
```diff
@@ -0,0 +1,5 @@
+package se.sundsvall.casedata.api.model.validation.enums;
+
+public enum MessageType {
```
Comment (AndreSoderlund) [inline src/main/java/se/sundsvall/casedata/api/model/validation/enums/MessageType.java:3]: Is there any drawbacks of reusing the MessageType enum in the "se.sundsvall.casedata.integration.db.model.enums" package?

Comment (cheezi0747) [reply src/main/java/se/sundsvall/casedata/api/model/validation/enums/MessageType.java:3]: Drawbacks? Maybe not. The intention was to move the enum to the validation package but seems like I just copied it. Will fix

File: src/main/java/se/sundsvall/casedata/integration/db/model/Message.java [hunk 2]
```diff
@@ -64,8 +63,7 @@ public class Message {
 
 	private String lastName;
 
-	@Enumerated(EnumType.STRING)
-	private MessageType messageType;
+	private String messageType;
```
[the remark this answers is not in the document]
Comment (cheezi0747) [reply src/main/java/se/sundsvall/casedata/integration/db/model/Message.java:66]: I don't see any real value in keeping this as a enum. That would mean a mapping 
string -> enum -> string and viceverse without no real benefit?

[the remark this answers is not in the document]
Comment (foobar-1337) [reply src/main/java/se/sundsvall/casedata/integration/db/model/Message.java:66]: Ok, I haven't looked at how this enum is used in this service.
Enums are often good for value restrictions, mappings, etc. (at least comparing to plain strings).

Bu if you don't see any value for an AttributeConverter here, then keep it as is.

**Review by AndreSoderlund — APPROVED** [review]

**Pull request merged**